simulation/CMakeLists.txt

```diff
@@ -2,11 +2,16 @@ set(SUBSYS_NAME simulation)
 set(SUBSYS_DESC "Point Cloud Library Simulation")
 set(SUBSYS_DEPS common io surface kdtree features search octree visualization filters geometry)
 
-set(build FALSE)
-find_package(OpenGL)
```

Comment:
Why was `OpenGL` removed?

Reply:
Its already searched for in main [CMakelist](https://github.com/PointCloudLibrary/pcl/blob/0bb69c0c795e852ddd731039cc1b0ac96b11d8ed/CMakeLists.txt#L466-L470):

But a guard checking if OPENGL_FOUND could be added indeed, like with GLEW - I'm not 100% sure, but I think GLEW requires Opengl and hence has to be available for GLEW to be found, so maybe this implicit check is suffiecient.